ui/server/src/main/scala/db/migration/V1_030__SpillStateToDisk.scala

```diff
@@ -0,0 +1,34 @@
+package db.migration
+
+import io.circe._
+import pl.touk.nussknacker.ui.db.migration.ProcessJsonMigration
+
+trait V1_030__SpillStateToDisk extends ProcessJsonMigration {
+
+  override def updateProcessJson(jsonProcess: Json): Option[Json] =
+    V1_030__SpillStateToDisk.renameSpillStateToDisk(jsonProcess)
+}
+
+object V1_030__SpillStateToDisk {
+
+  final val emptyAdditionalBranches = Json.fromValues(List.empty)
+
+  private[migration] def renameSpillStateToDisk(jsonProcess: Json): Option[Json] = {
+    val typeSpecificDataCursor = jsonProcess.hcursor.downField("metaData").downField("typeSpecificData")
+    val updatedTypeSpecificData = typeSpecificDataCursor
+      .withFocus(typeSpecificData => {
+        val spillStateToDisk = typeSpecificData.hcursor.downField("splitStateToDisk").focus
```

Comment:
IMO variable should be named `oldSpillStateToDisk` or `splitStateToDisk` because now it looks like you want to extract correct value instead of old one

Reply:
Done